symfony/plugins/orangehrmHelpPlugin/lib/processor/ZendeskHelpProcessor.php

```diff
@@ -0,0 +1,191 @@
+<?php
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+
+class ZendeskHelpProcessor implements HelpProcessor {
+
+    const DEFAULT_CONTENT_TYPE = "application/json";
+    const GET_METHOD = "GET";
+
+    protected $helpConfigService;
+
+    /**
+     * @return mixed
+     */
+    public function getHelpConfigService() {
+        if (!$this->helpConfigService instanceof HelpConfigService) {
+            $this->helpConfigService = new HelpConfigService();
+        }
+        return $this->helpConfigService;
+    }
+
+    /**
+     * @param mixed $helpConfigService
+     */
+    public function setHelpConfigService($helpConfigService) {
+        $this->helpConfigService = $helpConfigService;
+    }
+
+    public function getBaseUrl() {
+        return $this->getHelpConfigService()->getBaseHelpUrl();
+    }
+
+    public function getSearchUrlFromQuery($query=null,$labels=[],$categorieIds=[]) {
```

Comment:
Can't this manage via Guzzle, rather than string concatenation

Reply:
Checked with the documentation. seems not possible.